internal/configurator/testdata/curated-experience-test.yaml

```diff
@@ -0,0 +1,19 @@
+kubernetes:
+  jobs:
+    - job_name_prefix: kubernetes-job
```

Comment:
perhaps adding an extra job that will not be modified?

Reply:
we have the newrelic one already, let me see if I can add something to scrape it as well